terraform/modules/google-cloud/apps/relay/main.tf

```diff
@@ -203,7 +203,9 @@ resource "google_compute_instance_template" "application" {
     # NOTE: Minimum of 2 is required for XDP programs to load onto the NIC.
```

Comment:
Should this note be at the variable definition now?

Reply:
K, moved. Also added variable validations.